src/main/resources/io/jenkins/plugins/coverage/CoveragePublisher/config.jelly

```diff
@@ -2,9 +2,7 @@
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
 
     <f:block>
-        <f:entry title="Ant-Style Path for Automatically Detect" field="autoDetectPath">
-            <f:textbox default="*.xml" value="${instance.autoDetectPath}"/>
-        </f:entry>
+        <f:dropdownDescriptorSelector field="reportDetector" title="Report Detector" descriptors="${descriptor.listDetectorDescriptors}" />
```

Comment:
IIUC there should be another noop extension point implementation in such case

Reply:
noop extension? I don't understand....

Reply:
Oh, I know. You mean I should implement a noop detector?